bot/cogs/tags.py

```diff
@@ -113,7 +126,10 @@ def _get_tags_via_content(self, check: Callable[[Iterable], bool], keywords: str
 
         matching_tags = []
         for tag in self._cache.values():
-            if check(query in tag['embed']['description'].casefold() for query in keywords_processed):
+            if (
+                self.check_accessibility(user, tag)
+                and check(query in tag['embed']['description'].casefold() for query in keywords_processed)
+            ):
```

Comment:
Assigning the value to a separate variable avoids breaking the if statement into multiple lines.
```suggestion
            matches = (query in tag['embed']['description'].casefold() for query in keywords_processed)
            if self.check_accessibility(user, tag) and check(matches):
```